ovos_utils/gui.py

```diff
@@ -1,61 +1,96 @@
+from typing import Optional, Union
+
 import time
 from collections import namedtuple
 from enum import IntEnum
 from os.path import join
+# from ovos_bus_client import MessageBusClient
 
 from ovos_utils import resolve_ovos_resource_file, resolve_resource_file
 from ovos_utils.log import LOG
 from ovos_utils.messagebus import wait_for_reply, get_mycroft_bus, Message
 from ovos_utils.system import is_installed, has_screen, is_process_running
 
 
-def can_display():
-    return has_screen()
+def can_display() -> bool:
+    """
+    Return true if a display is available
+    """
+    return bool(has_screen())
 
 
-def is_gui_installed():
+def is_gui_installed() -> bool:
+    """
+    Return true if a GUI application is installed
+    """
     return is_installed("mycroft-gui-app") or \
            is_installed("ovos-shell") or \
            is_installed("mycroft-embedded-shell") or \
            is_installed("plasmashell")
 
 
-def is_gui_running():
+def is_gui_running() -> bool:
+    """
+    Return true if a GUI application is running
+    """
     return is_process_running("mycroft-gui-app") or \
```

Comment:
can we refactor this to future proof it a little, make the list of applications be an optional arg defaulting to the official gui clients, downstream may use different gui clients that this method will miss